crates/compiler/build/src/link.rs

```diff
@@ -143,7 +143,7 @@ pub fn build_zig_host_native(
 
     command.args(&[
         zig_host_src,
-        emit_bin,
+        &format!("-femit-bin={}", emit_bin),
```

Comment:
I moved this formatting to this function here, this seemed to be a be a more logical place compared to always having to do it before calling the function.